.github/workflows/release_js_sdk.yml

```diff
@@ -35,9 +38,14 @@ jobs:
       env:
         NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
 
+    - name: Update version
+      run: npm version ${{ github.event.inputs.version }} --no-git-tag-version
```

Comment:
The npm version command should include git configuration to ensure proper commit author:
```yaml
- name: Update version
  run: |
    git config --global user.email "github-actions@github.com"
    git config --global user.name "GitHub Actions"
    npm version ${{ github.event.inputs.version }} --no-git-tag-version
```